server/utils/nunjucksSetup.ts

```diff
@@ -13,7 +13,7 @@ export default function nunjucksSetup(app: express.Express, applicationInfo: App
   app.set('view engine', 'njk')
 
   app.locals.asset_path = '/assets/'
-  app.locals.applicationName = 'Sentence plan'
+  app.locals.applicationName = 'Sentence Plan'
```

Comment:
no, sorry this is a miscommunication. It is supposed to be "plan" - please revert this change.